sno/status.py

```diff
@@ -44,12 +44,13 @@ def get_status_json(repo):
 
 
 def get_branch_status_json(repo):
-    output = {"commit": None, "branch": None, "upstream": None}
+    output = {"commit": None, "abbrevCommit": None, "branch": None, "upstream": None}
```

Comment:
re terminology. upstream → remote?

Reply:
heh, in my review this comment was under that one ⇩

Reply:
Will change this in another PR. I think I need to change "schemaChanges" to "metaChanges" as well somewhere, for consistency.